autoload/writable_search/parser.vim

```diff
@@ -101,6 +111,23 @@ function! s:BuildProxies(grouped_lines)
   return proxies
 endfunction
 
+function! s:BuildProxiesFromQuickfix()
+  let proxies = []
+  let quickfix_list = getqflist()
+
+  for location in quickfix_list
+    let current_proxy = writable_search#proxy#New(location.bufnr)
```

Comment:
Hmm, it seems like this shouldn't work. If you're creating a new proxy in the loop, doesn't it end up with only one entry?